pennylane/io.py

```diff
@@ -562,7 +562,7 @@ def circuit():
             @qml.qnode(dev)
             def circuit():
                 mid_measure, *_ = loaded_circuit()
-                qml.cond(mid_measure == 0, qml.RX)(np.pi / 2, 0)
+                qml.cond(mid_measure == 0, qml.RX)(pnp.pi / 2, 0)
```

Comment:
```suggestion
                qml.cond(mid_measure == 0, qml.RX)(np.pi / 2, 0)
```

Is this necessary? I believe `np` is preferred for consistency in our documentation 👀

Reply:
Agreed, we use `pnp` for things that are meant to be trainable.